samples/default-authorizationserver/src/test/java/sample/DefaultAuthorizationServerApplicationTests.java

```diff
@@ -110,6 +118,37 @@ public void whenLoggingInAndRequestingTokenThenRedirectsToClientApplication() th
 		String location = response.getResponseHeaderValue("location");
 		assertThat(location).startsWith(REDIRECT_URI);
 		assertThat(location).contains("code=");
+
+
+		// ==============================================================================================
+		// The following token request should be performed by the client application,
+		// eg: a web application, a mobile app, etc.
+		// ==============================================================================================
+
+		// get code parameter value form location
+		String query = new URL(location).getQuery();
```

Comment:
The purpose of this test is to ensure the Authorization Response (with `code`) is redirected back to the client application after sending the Authorization Request. The Access Token request is not meant to be used in this test.